Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:8.10.2-jdk17 as build
+
+WORKDIR /app
+
+COPY . .
+
+RUN gradle bootjar
```

Comment:
I think you need to build the jar in this step

Reply:
done